### ISSUE TITLE
Wrong version on stats-page?

### ISSUE THREAD

**Issue by DigibitDK** [arut/nginx-rtmp-module#1650]
I'm pretty sure I'm using rtmp-module v.1.2.2, but the stats page says 1.1.4? In older versions as well ...

https://gyazo.com/69def65b3bb97dc342fab7bff9680696
(Generated by nginx-rtmp-module 1.1.4 ...)

Has this info just not been updated since v1.1.4? :D

**Comment by iphonelover** [comment]
邮件已经收到

**Comment by rrolla** [comment]
Yeah looks like version number it take from here: https://github.com/arut/nginx-rtmp-module/blob/master/ngx_rtmp_version.h#L12
